package.json

```diff
@@ -54,25 +54,25 @@
-    "test:extensions": "vitest run -r extensions --passWithNoTests --coverage && pnpm run test:extensions:compose && pnpm run test:extensions:kind && pnpm run test:extensions:docker && pnpm run test:extensions:lima && pnpm run test:extensions:kube && pnpm run test:extensions:podman && pnpm run test:extensions:registries && pnpm run test:extensions:kubectl-cli",
-    "test:extensions:kind": "vitest run -r extensions/kind --passWithNoTests --coverage ",
-    "test:extensions:compose": "vitest run -r extensions/compose --passWithNoTests --coverage",
-    "test:extensions:docker": "vitest run -r extensions/docker --passWithNoTests --coverage ",
-    "test:extensions:kube": "vitest run -r extensions/kube-context --passWithNoTests --coverage ",
-    "test:extensions:lima": "vitest run -r extensions/lima --passWithNoTests --coverage ",
-    "test:extensions:podman": "vitest run -r extensions/podman/packages/extension --passWithNoTests --coverage ",
-    "test:extensions:registries": "vitest run -r extensions/registries --passWithNoTests --coverage ",
-    "test:extensions:kubectl-cli": "vitest run -r extensions/kubectl-cli --passWithNoTests --coverage ",
-    "test:renderer": "pnpm run build:preload:types && pnpm run build:ui && vitest -c packages/renderer/vite.config.js run packages/renderer --passWithNoTests --coverage",
-    "test:ui": "vitest --config packages/ui/vite.config.js run --passWithNoTests --coverage",
-    "test:tools": "vitest run tools/src --passWithNoTests --coverage",
-    "test:scripts:stylesheet": "vitest run ./scripts/generate-stylesheet.spec.ts --passWithNoTests",
+    "test:main": "vitest --run --project=main --passWithNoTests",
+    "test:preload": "vitest --run --project=preload --passWithNoTests",
+    "test:preload-docker-extension": "vitest --run --project=preload-docker-extension --passWithNoTests",
+    "test:preload-webview": "vitest --run --project=preload-webview",
+    "test:extensions": "vitest --run --project=extensions --passWithNoTests && pnpm run test:extensions:compose && pnpm run test:extensions:kind && pnpm run test:extensions:docker && pnpm run test:extensions:lima && pnpm run test:extensions:kube && pnpm run test:extensions:podman && pnpm run test:extensions:registries && pnpm run test:extensions:kubectl-cli",
+    "test:extensions:kind": "vitest --run --project=kind --passWithNoTests",
+    "test:extensions:compose": "vitest --run --project=compose --passWithNoTests",
+    "test:extensions:docker": "vitest --run --project=docker --passWithNoTests",
+    "test:extensions:kube": "vitest --run --project=kube-context --passWithNoTests ",
+    "test:extensions:lima": "vitest --run --project=lima --passWithNoTests ",
+    "test:extensions:podman": "vitest --run --project=podman --passWithNoTests ",
+    "test:extensions:registries": "vitest --run --project=registries --passWithNoTests ",
+    "test:extensions:kubectl-cli": "vitest --run --project=kubectl-cli --passWithNoTests",
```

Comment:
as a follow-up, I'm wondering if we should not name project with `extension/` prefix

like `--project=extension/kind`

Reply:
Oh yeah, didn't think it would work